docs/ss_slide_000.md

```diff
@@ -13,4 +13,4 @@
 * [Supporting Information](https://rivesunder.github.io/yuca)
 
 {:style="text-align:center;"}
-[Supporting resources](https://rivesunder.github.io/yuca) -- [Next slide](https://rivesunder.github.io/yuca/ss_slide_001)
+[Supporting resources](https://rivesunder.github.io/yuca) -- [Next slide](https://rivesunder.github.io/yuca_docs/ss_slide_001)
```

Comment:
Let's go ahead and revert the links to the gh-pages to point to `https://rivesunder.github.io/yuca` instead of yuca_docs. This should decrease navigation confusion while still offloading the larger assets storage to yuca_docs.